### ISSUE TITLE
iframe not rendering newly introduced rounded corners nicely on Safari 15.3

### ISSUE THREAD

**Issue by saschaludwig** [home-assistant/frontend#14272]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

The new rounded corners are not beeing rendered nicely when using the "iframe" card type. Referred in the documentation as “webpage card” (https://www.home-assistant.io/dashboards/iframe/)
The iframe is simply on top, making the rounded corners look ugly in Safari. However in Chrome and Firefox it looks okay.

### Describe the behavior you expected

iframe is rendered below the rounded corners / bounding box around the iframe with rounded corners

### Steps to reproduce the issue

1. add an iframe panel with website to a dashboard
![Bildschirmfoto 2022-11-03 um 15 24 32](https://user-images.githubusercontent.com/12997980/199751197-8d60ae14-5d42-4ef5-a8c8-4468184b65f6.png)

### What version of Home Assistant Core has the issue?

2022.11.0

### In which browser are you experiencing the issue with?

Safari 15.3

### Which operating system are you using to run this browser?

macOS Monterey 12.2.1

### Problem-relevant frontend configuration

```yaml
type: iframe
url: >-
  https://grafana.xxxxxxxxxxxx/d-solo/xxxx/xxxx
aspect_ratio: 100%
```

**Comment by saschaludwig** [comment]
Still not fixed in 2023.1.4

**Comment by saschaludwig** [comment]
Still not fixed in 2023.4.6

**Comment by saschaludwig** [comment]
Still not fixed in 2023.8.4

**Comment by ildar170975** [comment]
Check this:
![изображение](https://github.com/home-assistant/frontend/assets/71872483/c81b10c2-1dab-4064-8b5e-21b2504a7cf5)
```
type: iframe
url: https://yahoo.com
aspect_ratio: 100%
```
Rounded corners are here.
